wrappers/src/shared_realm_cs.cpp

```diff
@@ -369,6 +374,41 @@ REALM_EXPORT uint64_t shared_realm_get_schema_version(SharedRealm& realm, Native
     });
 }
 
+REALM_EXPORT uint32_t shared_realm_begin_transaction_async(SharedRealm& realm, void* tcs_ptr, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        return realm->async_begin_transaction([tcs_ptr, ex]()
+        {
+            s_handle_write_commit_async(tcs_ptr, ex);
+        }, true);
+    });
+}
+
+REALM_EXPORT uint32_t shared_realm_commit_transaction_async(SharedRealm& realm, void* tcs_ptr, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        return realm->async_commit_transaction([tcs_ptr](std::exception_ptr err) {
+            NativeException::Marshallable nativeEx { RealmErrorType::NoError };
+            if (err) {
+                nativeEx = convert_exception(err).for_marshalling();
+            }
+            s_handle_write_commit_async(tcs_ptr, nativeEx);
+        }, false);
```

Comment:
We don't need `false` here

Reply:
This must be on a similar note as the comment above. Is this right?

Reply:
Yes - if we're providing explicit argument values, we should name the arguments. As a reader, I don't know what `false` means in this case, so it's more harmful than helpful.